Parsers/ASimFileEvent/ProductParsers/vimFileEventMicrosoftAuditEvent.yaml

```diff
@@ -1,106 +1,99 @@
-      , "0x40", "ObjectDeleted"
-      , "0x40000", "MetadataModified"
-      , "0x6", "ObjectModified"
-      , "0x8", "MetadataAccessed"
-      , "0x80", "MetadataAccessed"
-      , "0x80000", "MetadataModified"
-  ];
-  let UserTypeLookup = datatable (AccountType:string, ActorUserType:string)
-  [
-    'User', 'Regular',
-    'Machine', 'Machine'
-  ];    
-  let KnownSIDs = datatable (sid:string, username:string, type:string)
-  [
-    'S-1-5-18', 'Local System', 'Simple',
-    'S-1-0-0', 'Nobody', 'Simple'
-  ];
-  union isfuzzy=false (WindowsEvent
-  | where not(disabled)  
-  | where EventID == 4663 
-    and EventData.ObjectType == "File"
-    and EventData.ObjectName !startswith @"\Device\"
-  | project TimeGenerated
-    , EventID, AccessMask = tostring(EventData.AccessMask)
-    , ProcessName = tostring(EventData.ProcessName)
-    , SubjectUserSid = tostring(EventData.SubjectUserSid)
-    , AccountType = tostring(EventData.AccountType)
-    , Computer = tostring(EventData.Computer)
-    , ObjectName = tostring(EventData.ObjectName)
-    , ProcessId = tostring(EventData.ProcessId)
-    , SubjectUserName = tostring(EventData.SubjectUserName)
-    , SubjectAccount = tostring(EventData.SubjectAccount)
-    , SubjectLogonId = tostring(EventData.SubjectLogonId)
-    , HandleId = tostring(EventData.HandleId)
-  )
-  , (SecurityEvent
-  | where not(disabled)  
-  | where EventID == 4663 
-    and ObjectType == "File"
-    and ObjectName !startswith @"\Device\"
-  | project TimeGenerated, EventID, AccessMask, ProcessName, SubjectUserSid, AccountType, Computer, ObjectName, ProcessId, SubjectUserName, SubjectAccount, SubjectLogonId, HandleId)
-  | lookup EventTypeLookup on AccessMask
-  | lookup UserTypeLookup on AccountType
-  | lookup KnownSIDs on $left.SubjectUserSid == $right.sid
-  | extend ActingProcessName = ProcessName
-    , ActorUsername = iff (SubjectUserName == "-", username, SubjectAccount)
-    , ActorUsernameType = iff(SubjectUserName == '-',type, 'Windows')
-    , EventStartTime = TimeGenerated
-    , EventEndTime = TimeGenerated
-    , TargetFilePath = ObjectName
-    , TargetFilePathFormat = "Windows Local"
-    , ActingProcessId = tostring(toint(ProcessId))
-    , EventOriginalType = tostring(EventID)
-  | project-away EventID, ProcessId, AccountType, type, username
-  | project-rename ActorUserId = SubjectUserSid
-    , DvcHostname = Computer
-    , Process = ProcessName
-    , FilePath = ObjectName
-    , ActorSessionId = SubjectLogonId
-    , FileSessionId = HandleId
-  | extend EventSchema = "FileEvent"
-    , EventSchemaVersion = "0.1.1"
-    , EventResult = "Success"
-    , EventCount = int(1)
-    , EventVendor = 'Microsoft'
-    , EventProduct = 'Security Events'
-    , Dvc = DvcHostname
-    , ActorWindowsUsername = ActorUsername
-    , User = ActorUsername
-    , ActorUserSid = ActorUserId
-  };
-  Parser (disabled)
\ No newline at end of file
+Parser:
+  Title: Microsoft File Storage - Eventlog - Audit Events
```

Comment:
Update Title, Product, Description and ParserName, and the file name itself according to [this](
https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/ASimNetworkSession/Parsers/vimNetworkSessionMicrosoftWindowsEventFirewall.yaml). The texts should be the same with the needed changes. The ParserName should be vimFileEventMicrosoftWindowsEvents

If you prefer me to do that after we merge, this is also find.